src/topp/OMSSAAdapter.cpp

```diff
@@ -201,8 +201,9 @@ class TOPPOMSSAAdapter :
     registerOutputFile_("out", "<file>", "", "Output file ");
     setValidFormats_("out", ListUtils::create<String>("idXML"));
 
-    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 1.5, "Precursor mass tolerance (Default: Dalton)", false);
-    registerFlag_("precursor_mass_tolerance_unit_ppm", "If this flag is set, ppm is used as precursor mass tolerance unit");
+    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 10.0, "Precursor mass tolerance (Default: Dalton)", false);
```

Comment:
In all the other descriptions you specifically mention that it is the monoisotopic mass that is specified here.